atlas_scripts/squirrel_monkey.py

```diff
@@ -0,0 +1,572 @@
+        if ref_path.exists():
+            reference = load_any(ref_path, as_numpy=True)
+            print(f"Loaded reference from: {ref_path}")
+            break
+
+    if reference is None:
+        # Try to find any template file
+        template_files = list(download_dir_path.rglob("*template*.nii*"))
+        if template_files:
+            reference = load_any(template_files[0], as_numpy=True)
+            print(f"Loaded reference from: {template_files[0]}")
+        else:
+            raise FileNotFoundError(
+                "Could not find reference template. "
+                "Please check file structure in downloaded archive."
+            )
+
+    # Load annotation/labels
+    possible_annotation_paths = [
+        download_dir_path / "VALiDATe29" / "labels.nii.gz",
+        download_dir_path / "VALiDATe29" / "labels.nii",
+        download_dir_path / "VALiDATe29" / "atlas_labels.nii.gz",
+        download_dir_path / "VALiDATe29" / "segmentation.nii.gz",
+        download_dir_path / "labels.nii.gz",
+        download_dir_path / "labels.nii",
+    ]
+
+    annotation = None
+    for annot_path in possible_annotation_paths:
+        if annot_path.exists():
+            annotation = load_any(annot_path, as_numpy=True).astype(np.int64)
+            print(f"Loaded annotation from: {annot_path}")
+            break
+
+    if annotation is None:
+        # Try to find any label file
+        label_files = list(download_dir_path.rglob("*label*.nii*"))
+        if label_files:
+            annotation = load_any(label_files[0], as_numpy=True).astype(
+                np.int64
+            )
+            print(f"Loaded annotation from: {label_files[0]}")
+        else:
+            print(
+                "Warning: No annotation file found. Creating basic annotation."
+            )
+            annotation = np.zeros_like(reference, dtype=np.int64)
+            annotation[reference > 0] = 1  # Simple binary mask
+
+    return reference, annotation
+
+
+def retrieve_hemisphere_map():
+    """
+    Retrieve a hemisphere map for the atlas.
+
+    Returns None to indicate symmetrical atlas, allowing automatic
+    hemisphere generation.
+
+    Returns
+    -------
+    np.ndarray or None
+        None, indicating the atlas should be treated as symmetrical.
+    """
+    return None
+
+
+def retrieve_structure_information(download_dir_path):
+    """
+    Return a list of dictionaries with information about brain structures.
+
+    The VALiDATe29 atlas includes 81 labels:
+    - 18 cortical gray matter regions
+    - 57 white matter tracts
+    - 6 ventricle labels
+
+    Parameters
+    ----------
+    download_dir_path : pathlib.Path
+        The directory containing the downloaded atlas files.
+
+    Returns
+    -------
+    list[dict]
+        A list of dictionaries, each containing information for a single
+        atlas structure.
+    """
+    # Basic structure list - root + brain
+    structures = [
```

Comment:
I may be wrong but I believe usually the root is the brain and does not require two different IDs